scanner/image/db/scripts/init-entrypoint.sh

```diff
@@ -1,8 +1,54 @@
 #!/usr/bin/env bash
 
+# init-entrypoint.sh initializes the DB if it does not exist.
+
 set -Eeo pipefail
 
-# Initialize DB if it does not exist
-if [ ! -s "$PGDATA/PG_VERSION" ]; then
-  initdb --auth-host="$POSTGRES_HOST_AUTH_METHOD" --auth-local="$POSTGRES_HOST_AUTH_METHOD" --pwfile "$POSTGRES_PASSWORD_FILE" --data-checksums
+# load-init-bundle calls pg_restore on the database init bundle.
+load-init-bundle() {
+    local db=${1:?missing first required argument: db}
+    local bundle=${2:?missing second required argument: f}
+    local dump=$(mktemp)
+
+    zstd >$dump -dc $bundle
+    pg_ctl start
+    pg_restore \
+        --verbose \
+        --format=custom \
+        --jobs $(nproc) \
+        --exit-on-error \
+        --dbname $db \
+        --no-owner \
+        $dump
+    pg_stop
```

Comment:
Lot's of potential optimizations here: https://www.postgresql.org/docs/current/populate.html